### PR TITLE
Add lifecycle information to the site

### PR DESCRIPTION
**Pull request by HoverBaum**
As a result of dicussion in #19 and following up on #22 as well as some general dicussion lately this pull request aimes to add more information regarding what repostatus is about and which status is appropriet for a repo tot he website.

To help people choose a status for the repo and to help them understand what repo status wants to talk about a "lifecycle" page is added. This page puts statuses in relationshiops to each other. We think about a status as a state a repository is in and talk about when and how a state should change.

lifecycle

## Needs dicussion:

- I am not sure calling this "Lifecycle" is the best thing we could come up with. Maybe this should be a more general page talking about the conceirns as outlined above. But so far thisw as my best idea for it.
- Input on the actual relations between states would be greatly appreciated. Please refer to the image above as well as the text in the pull request.

### CODE REVIEW

**Comment by jantman** [comment]
@HoverBaum thanks so much for this, and sorry for not responding sooner!

This looks really good to me. There are a few minor spelling issues which I commented on; I certainly do not mean that to be negative at all; I've always had major difficulties with other languages and honestly what you have here is as good as I'd expect from someone who lived their entire life writing in only English.

I think the only two things I'd ask are whether you could also include the source file for the diagram, just for completeness, and whether you could add in the new "Moved" status?

The relations in the diagram look fine to me. I don't see anything obviously "wrong" or missing.

As to "Lifecycle"... I can't think of a better word or name. That's probably what I would have chosen as well.

File: gh_pages/lifecycle.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+layout: minimal
+---
+
+## Lifecycle
+
+You can think of respostatus as a way to describe the lifecycle-phase of a project. This may help you to choose an appropriate status for your repositories.
+
+Repostatus aimes to answer two questions about a repository:
```
Comment (jantman) [inline gh_pages/lifecycle.md]: "aimes" -> "aims"

File: gh_pages/lifecycle.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+* How complete is this? Should I expect it to be "stable" software, mostly working, or not really useful at all?
+* Should I expect active development and support, maybe some bug fixes, or nothing at all?
+
+We wish to talk about whether the content of a repository is in a usable state and how much support from contributors can be expected. Thus statuses can be divided along two metrics: usableness and support.
```
Comment (jantman) [inline gh_pages/lifecycle.md]: I think that "axes" (the plural of axis) would be more idiomatic English. I'd probably also go with "usability".

File: gh_pages/lifecycle.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+![Flowdiagram representation of a typical lifecycle.](/images/lifecycle.png)
+
+As illustrated above the statuses of a repository can be split into two groups: unstable repos and stable repos.
+
+### Unstable repos
+
+These repositories have not yet reached a stable and usable state. You should expect them to change frequently and dramatically.
+
+All repositories start as a Concept. They are little more than an idea yet. From here we might abandone the project or start to work on it in ernest. Only once a repository was a WIP can it be suspended, work on it be put aside t be continued later. To reach a stable state some work is needed thus WIP is the only state from which we can feasibly move towards a stable state.
```
Comment (jantman) [inline gh_pages/lifecycle.md]: "abandone" -> "abandon"

Also, I think I'd change this to "All projects start as a Concept", just to account for the people who (scary, but some people do) start a project without version control.

Also looks like a typo, "put aside t be", "o" missing.

File: gh_pages/lifecycle.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+### Stable repos
+
+A repository will reach a stable state as something that is actively worked on. Thus all repositories will move into the second group of states as Active. From here they might become Inactive or Unsupported. Contrary to the unstable states it is totally feasable for a repository to move from any stable state into any other stable state.
+
+### Moved
+
+A special case is the *Moved* state for repositories. Any repository can move to be *Moved* from any state. As this states indicates that another repository will contain the actual state of the project.
```
Comment (jantman) [inline gh_pages/lifecycle.md]: I think I would add an example here to make it more clear. Maybe add to the end of this sentence: ", such as when a repository is moved to a different hosting provider or if the maintainer stops working on it and makes a fork the new authoritative version." Something like that.

File: gh_pages/lifecycle.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+## Real life application
+
+Understanbly many repositories will not stricktly follow the lifecycle outlined here. When a few commits move a repo from *Suspended* to *Active* it is understandable that maintners might not want to make another commit just to move it to *WIP*.
```
Comment (jantman) [inline gh_pages/lifecycle.md]: "stricktly" -> "strictly"

File: gh_pages/lifecycle.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+
+We fully understand that states as outlined here will be skipped. Therefore the lifecycle hopes to outline a way of thinking about the status of a repository and help choose an appropriet status.
```
Comment (jantman) [inline gh_pages/lifecycle.md]: "appropriet" -> "appropriate"

**Comment by HoverBaum** [comment]
@jantman thanks for your input. I also took way longer to get this going than I intended to.

I will include your spelling corrections. As for the source file I can happily include that.

As to the "moved" status I am unsure if it would be wise to include that in the lifecylce image above. As it would be a state and an arrow from every other state to the "moved" one. I feel like this would add unneded complexity and might obscure the other relationships instead of adding value. That said it might work to include it besides the other states without arrows from each one and indicate somehow that it can be reached from all states.

Currently I would favor to add a second image like this one: 

moved-diagram

**Comment by jantman** [comment]
Ok, thanks!

You're right, after thinking about it a bit more and reading your thoughts, I agree that Moved shouldn't be added in the lifecycle diagram. Maybe it should just be left out of the diagrams completely and only mentioned in the text... like you originally did!

**Comment by HoverBaum** [comment]
@jantman okay then I will go with that and I will see if I can maybe make that reasoning clearer for the site.

File: gh_pages/lifecycle.md (hunk 1 above)
Comment (HoverBaum) [reply gh_pages/lifecycle.md]: ✔️

File: gh_pages/lifecycle.md (hunk 2 above)
Comment (HoverBaum) [reply gh_pages/lifecycle.md]: ✔️

File: gh_pages/lifecycle.md (hunk 3 above)
Comment (HoverBaum) [reply gh_pages/lifecycle.md]: ✔️

File: gh_pages/lifecycle.md (hunk 4 above)
Comment (HoverBaum) [reply gh_pages/lifecycle.md]: ✔️

File: gh_pages/lifecycle.md (hunk 5 above)
Comment (HoverBaum) [reply gh_pages/lifecycle.md]: ✔️

File: gh_pages/lifecycle.md (hunk 6 above)
Comment (HoverBaum) [reply gh_pages/lifecycle.md]: ✔️

**Comment by HoverBaum** [comment]
@jantman finally picked this back up :)

Not sure how I can provide the image nicely. Used some online tooling for that that doesn't have any nice exports. Switched to an SVG though. Hope that people might be easier able to edit that.

**Comment by waldyrious** [comment]
In case it helps, here is the SVG source cleaned up by [SVGOMG](https://jakearchibald.github.io/svgomg/):

```svg
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:lucid="lucid" width="792" height="1080">
  <g transform="translate(-489 -220)" lucid:page-tab-id="0_0">
    <path d="M710 270c0 16.57-13.43 30-30 30s-30-13.43-30-30 13.43-30 30-30 30 13.43 30 30z" stroke="#666" stroke-width="2" fill="#fff"/>
    <path d="M680 301v39.76" stroke="#999" stroke-width="2" fill="none"/>
    <path d="M680 300l1-.04V301h-2v-1.08zm0 55.76l-4.64-14.26h9.28z" fill="#999"/>
    <path d="M680 359l-6-18.5h12zm-3.26-16.5l3.26 10.03 3.26-10.03z" fill="#999"/>
    <path d="M600 365c0-2.76 2.24-5 5-5h150c2.76 0 5 2.24 5 5v110c0 2.76-2.24 5-5 5H605c-2.76 0-5-2.24-5-5z" stroke="#666" stroke-width="2" fill="#fff"/>
    <use xlink:href="#a" transform="translate(646.53 423.15)"/>
    <path d="M761 420h169.76" stroke="#999" stroke-width="2" fill="none"/>
    <path d="M761 421h-1v-2h1zm184.76-1l-14.26 4.64v-9.28z" fill="#999"/>
    <path d="M949 420l-18.5 6v-12zm-16.5 3.26l10.03-3.26-10.03-3.26z" fill="#999"/>
    <path d="M950 365c0-2.76 2.24-5 5-5h150c2.76 0 5 2.24 5 5v110c0 2.76-2.24 5-5 5H955c-2.76 0-5-2.24-5-5z" stroke="#666" stroke-width="2" fill="#fff"/>
    <use xlink:href="#b" transform="translate(1013.03 423.15)"/>
    <path d="M1030 481v129.76" stroke="#999" stroke-width="2" fill="none"/>
    <path d="M1031 481h-2v-1h2zm-1 144.76l-4.64-14.26h9.28z" fill="#999"/>
    <path d="M1030 629l-6-18.5h12zm-3.26-16.5l3.26 10.03 3.26-10.03z" fill="#999"/>
    <path d="M950 635c0-2.76 2.24-5 5-5h150c2.76 0 5 2.24 5 5v110c0 2.76-2.24 5-5 5H955c-2.76 0-5-2.24-5-5z" stroke="#666" stroke-width="2" fill="#fff"/>
    <use xlink:href="#c" transform="translate(984.505 693.15)"/>
    <path d="M949 690H779.24" stroke="#999" stroke-width="2" fill="none"/>
    <path d="M950 691h-1v-2h1zm-171.5 3.64L764.24 690l14.26-4.64z" fill="#999"/>
    <path d="M779.5 696l-18.5-6 18.5-6zm-12.03-6l10.03 3.26v-6.52z" fill="#999"/>
    <path d="M600 635c0-2.76 2.24-5 5-5h150c2.76 0 5 2.24 5 5v110c0 2.76-2.24 5-5 5H605c-2.76 0-5-2.24-5-5z" stroke="#666" stroke-width="2" fill="#fff"/>
    <use xlink:href="#d" transform="translate(634.005 693.15)"/>
    <path d="M680 481v129.76" stroke="#999" stroke-width="2" fill="none"/>
    <path d="M681 481h-2v-1h2zm-1 144.76l-4.64-14.26h9.28z" fill="#999"/>
    <path d="M680 629l-6-18.5h12zm-3.26-16.5l3.26 10.03 3.26-10.03z" fill="#999"/>
    <path d="M1069.33 629V499.24" stroke="#999" stroke-width="2" fill="none"/>
    <path d="M1070.33 630h-2v-1h2zm3.64-131.5h-9.27l4.63-14.26z" fill="#999"/>
    <path d="M1075.35 499.5h-12.03l6-18.5zm-9.28-2h6.52l-3.27-10.03z" fill="#999"/>
    <path d="M716 629v-49h104V456h110.76" stroke="#999" stroke-width="2" fill="none"/>
    <path d="M717 630h-2v-1h2zm228.76-174l-14.26 4.64v-9.28z" fill="#999"/>
    <path d="M949 456l-18.5 6v-12zm-16.5 3.26l10.03-3.26-10.03-3.26z" fill="#999"/>
    <path d="M982.67 481v49H890v126H779.24" stroke="#999" stroke-width="2" fill="none"/>
    <path d="M983.67 481h-2v-1h2zM778.5 660.64L764.24 656l14.26-4.64z" fill="#999"/>
    <path d="M779.5 662l-18.5-6 18.5-6zm-12.03-6l10.03 3.26v-6.52z" fill="#999"/>
    <path d="M1111 420h40v439h-121v21.76" stroke="#999" stroke-width="2" fill="none"/>
    <path d="M1111 421h-1v-2h1zm-81 474.76l-4.64-14.26h9.28z" fill="#999"/>
    <path d="M1030 899l-6-18.5h12zm-3.26-16.5l3.26 10.03 3.26-10.03z" fill="#999"/>
    <path d="M950 905c0-2.76 2.24-5 5-5h150c2.76 0 5 2.24 5 5v110c0 2.76-2.24 5-5 5H955c-2.76 0-5-2.24-5-5z" stroke="#666" stroke-width="2" fill="#fff"/>
    <use xlink:href="#e" transform="translate(1005.53 963.15)"/>
    <path d="M949 960H779.24" stroke="#999" stroke-width="2" fill="none"/>
    <path d="M950 961h-1v-2h1zm-171.5 3.64L764.24 960l14.26-4.64z" fill="#999"/>
    <path d="M779.5 966l-18.5-6 18.5-6zm-12.03-6l10.03 3.26v-6.52z" fill="#999"/>
    <path d="M600 905c0-2.76 2.24-5 5-5h150c2.76 0 5 2.24 5 5v110c0 2.76-2.24 5-5 5H605c-2.76 0-5-2.24-5-5z" stroke="#666" stroke-width="2" fill="#fff"/>
    <use xlink:href="#f" transform="translate(628.555 963.15)"/>
    <path d="M1030 1021v119.76" stroke="#999" stroke-width="2" fill="none"/>
    <path d="M1031 1021h-2v-1h2zm-1 134.76l-4.64-14.26h9.28z" fill="#999"/>
    <path d="M1030 1159l-6-18.5h12zm-3.26-16.5l3.26 10.03 3.26-10.03z" fill="#999"/>
    <path d="M950 1165c0-2.76 2.24-5 5-5h150c2.76 0 5 2.24 5 5v110c0 2.76-2.24 5-5 5H955c-2.76 0-5-2.24-5-5z" stroke="#666" stroke-width="2" fill="#fff"/>
    <use xlink:href="#g" transform="translate(999.03 1223.15)"/>
    <path d="M680 1021v199h250.76" stroke="#999" stroke-width="2" fill="none"/>
    <path d="M681 1021h-2v-1h2zm264.76 199l-14.26 4.64v-9.28z" fill="#999"/>
    <path d="M949 1220l-18.5 6v-12zm-16.5 3.26l10.03-3.26-10.03-3.26z" fill="#999"/>
    <path d="M1069.33 1159v-119.76" stroke="#999" stroke-width="2" fill="none"/>
    <path d="M1070.33 1160h-2v-1h2zm3.64-121.5h-9.27l4.63-14.26z" fill="#999"/>
    <path d="M1075.35 1039.5h-12.03l6-18.5zm-9.28-2h6.52l-3.27-10.03z" fill="#999"/>
    <path d="M761 992h169.76" stroke="#999" stroke-width="2" fill="none"/>
    <path d="M761 993h-1v-2h1zm184.76-1l-14.26 4.64v-9.28z" fill="#999"/>
    <path d="M949 992l-18.5 6v-12zm-16.5 3.26l10.03-3.26-10.03-3.26z" fill="#999"/>
    <path d="M949 1189.33H713.33v-150.1" stroke="#999" stroke-width="2" fill="none"/>
    <path d="M950 1190.33h-1v-2h1zM717.97 1038.5h-9.27l4.63-14.26z" fill="#999"/>
    <path d="M719.35 1039.5h-12.03l6-18.5zm-9.28-2h6.52l-3.27-10.03z" fill="#999"/>
    <path d="M726.58 812H510v-4h216.58zm533.42 0H985.42v-4H1260z" fill="#c7e8ac"/>
    <use xlink:href="#h" transform="translate(726.58 813.6)"/>
    <use xlink:href="#i" transform="translate(774.53 813.6)"/>
    <use xlink:href="#j" transform="translate(808.53 813.6)"/>
    <use xlink:href="#k" transform="translate(878.48 813.6)"/>
    <use xlink:href="#l" transform="translate(893.48 813.6)"/>
    <use xlink:href="#m" transform="translate(946.43 813.6)"/>
    <defs>
      <path fill="#333" d="M212-179c-10-28-35-45-73-45-59 0-87 40-87 99 0 60 29 101 89 101 43 0 62-24 78-52l27 14C228-24 195 4 139 4 59 4 22-46 18-125c-6-104 99-153 187-111 19 9 31 26 39 46" id="n"/>
      <path fill="#333" d="M100-194c62-1 85 37 85 99 1 63-27 99-86 99S16-35 15-95c0-66 28-99 85-99zM99-20c44 1 53-31 53-75 0-43-8-75-51-75s-53 32-53 75 10 74 51 75" id="o"/>
      <path fill="#333" d="M117-194c89-4 53 116 60 194h-32v-121c0-31-8-49-39-48C34-167 62-67 57 0H25l-1-190h30c1 10-1 24 2 32 11-22 29-35 61-36" id="p"/>
      <path fill="#333" d="M96-169c-40 0-48 33-48 73s9 75 48 75c24 0 41-14 43-38l32 2c-6 37-31 61-74 61-59 0-76-41-82-99-10-93 101-131 147-64 4 7 5 14 7 22l-32 3c-4-21-16-35-41-35" id="q"/>
      <path fill="#333" d="M100-194c63 0 86 42 84 106H49c0 40 14 67 53 68 26 1 43-12 49-29l28 8c-11 28-37 45-77 45C44 4 14-33 15-96c1-61 26-98 85-98zm52 81c6-60-76-77-97-28-3 7-6 17-6 28h103" id="r"/>
      <path fill="#333" d="M115-194c55 1 70 41 70 98S169 2 115 4C84 4 66-9 55-30l1 105H24l-1-265h31l2 30c10-21 28-34 59-34zm-8 174c40 0 45-34 45-75s-6-73-45-74c-42 0-51 32-51 76 0 43 10 73 51 73" id="s"/>
      <path fill="#333" d="M59-47c-2 24 18 29 38 22v24C64 9 27 4 27-40v-127H5v-23h24l9-43h21v43h35v23H59v120" id="t"/>
      <g id="a">
        <use transform="scale(.05)" xlink:href="#n"/>
        <use transform="translate(12.95) scale(.05)" xlink:href="#o"/>
        <use transform="translate(22.95) scale(.05)" xlink:href="#p"/>
        <use transform="translate(32.95) scale(.05)" xlink:href="#q"/>
        <use transform="translate(41.95) scale(.05)" xlink:href="#r"/>
        <use transform="translate(51.95) scale(.05)" xlink:href="#s"/>
        <use transform="translate(61.95) scale(.05)" xlink:href="#t"/>
      </g>
      <path fill="#333" d="M266 0h-40l-56-210L115 0H75L2-248h35L96-30l15-64 43-154h32l59 218 59-218h35" id="u"/>
      <path fill="#333" d="M33 0v-248h34V0H33" id="v"/>
      <path fill="#333" d="M30-248c87 1 191-15 191 75 0 78-77 80-158 76V0H30v-248zm33 125c57 0 124 11 124-50 0-59-68-47-124-48v98" id="w"/>
      <g id="b">
        <use transform="scale(.05)" xlink:href="#u"/>
        <use transform="translate(16.95) scale(.05)" xlink:href="#v"/>
        <use transform="translate(21.95) scale(.05)" xlink:href="#w"/>
      </g>
      <path fill="#333" d="M185-189c-5-48-123-54-124 2 14 75 158 14 163 119 3 78-121 87-175 55-17-10-28-26-33-46l33-7c5 56 141 63 141-1 0-78-155-14-162-118-5-82 145-84 179-34 5 7 8 16 11 25" id="x"/>
      <path fill="#333" d="M84 4C-5 8 30-112 23-190h32v120c0 31 7 50 39 49 72-2 45-101 50-169h31l1 190h-30c-1-10 1-25-2-33-11 22-28 36-60 37" id="y"/>
      <path fill="#333" d="M135-143c-3-34-86-38-87 0 15 53 115 12 119 90S17 21 10-45l28-5c4 36 97 45 98 0-10-56-113-15-118-90-4-57 82-63 122-42 12 7 21 19 24 35" id="z"/>
      <path fill="#333" d="M85-194c31 0 48 13 60 33l-1-100h32l1 261h-30c-2-10 0-23-3-31C134-8 116 4 85 4 32 4 16-35 15-94c0-66 23-100 70-100zm9 24c-40 0-46 34-46 75 0 40 6 74 45 74 42 0 51-32 51-76 0-42-9-74-50-73" id="A"/>
      <g id="c">
        <use transform="scale(.05)" xlink:href="#x"/>
        <use transform="translate(12) scale(.05)" xlink:href="#y"/>
        <use transform="translate(22) scale(.05)" xlink:href="#z"/>
        <use transform="translate(31) scale(.05)" xlink:href="#s"/>
        <use transform="translate(41) scale(.05)" xlink:href="#r"/>
        <use transform="translate(51) scale(.05)" xlink:href="#p"/>
        <use transform="translate(61) scale(.05)" xlink:href="#A"/>
        <use transform="translate(71) scale(.05)" xlink:href="#r"/>
        <use transform="translate(81) scale(.05)" xlink:href="#A"/>
      </g>
      <path fill="#333" d="M205 0l-28-72H64L36 0H1l101-248h38L239 0h-34zm-38-99l-47-123c-12 45-31 82-46 123h93" id="B"/>
      <path fill="#333" d="M115-194c53 0 69 39 70 98 0 66-23 100-70 100C84 3 66-7 56-30L54 0H23l1-261h32v101c10-23 28-34 59-34zm-8 174c40 0 45-34 45-75 0-40-5-75-45-74-42 0-51 32-51 76 0 43 10 73 51 73" id="C"/>
      <path fill="#333" d="M141-36C126-15 110 5 73 4 37 3 15-17 15-53c-1-64 63-63 125-63 3-35-9-54-41-54-24 1-41 7-42 31l-33-3c5-37 33-52 76-52 45 0 72 20 72 64v82c-1 20 7 32 28 27v20c-31 9-61-2-59-35zM48-53c0 20 12 33 32 33 41-3 63-29 60-74-43 2-92-5-92 41" id="D"/>
      <g id="d">
        <use transform="scale(.05)" xlink:href="#B"/>
        <use transform="translate(12) scale(.05)" xlink:href="#C"/>
        <use transform="translate(22) scale(.05)" xlink:href="#D"/>
        <use transform="translate(32) scale(.05)" xlink:href="#p"/>
        <use transform="translate(42) scale(.05)" xlink:href="#A"/>
        <use transform="translate(52) scale(.05)" xlink:href="#o"/>
        <use transform="translate(62) scale(.05)" xlink:href="#p"/>
        <use transform="translate(72) scale(.05)" xlink:href="#r"/>
        <use transform="translate(82) scale(.05)" xlink:href="#A"/>
      </g>
      <path fill="#333" d="M24-231v-30h32v30H24zM24 0v-190h32V0H24" id="E"/>
      <path fill="#333" d="M108 0H70L1-190h34L89-25l56-165h34" id="F"/>
      <g id="e">
        <use transform="scale(.05)" xlink:href="#B"/>
        <use transform="translate(12) scale(.05)" xlink:href="#q"/>
        <use transform="translate(21) scale(.05)" xlink:href="#t"/>
        <use transform="translate(26) scale(.05)" xlink:href="#E"/>
        <use transform="translate(29.95) scale(.05)" xlink:href="#F"/>
        <use transform="translate(38.95) scale(.05)" xlink:href="#r"/>
      </g>
      <path fill="#333" d="M232-93c-1 65-40 97-104 97C67 4 28-28 28-90v-158h33c8 89-33 224 67 224 102 0 64-133 71-224h33v155" id="G"/>
      <path fill="#333" d="M114-163C36-179 61-72 57 0H25l-1-190h30c1 12-1 29 2 39 6-27 23-49 58-41v29" id="H"/>
      <g id="f">
        <use transform="scale(.05)" xlink:href="#G"/>
        <use transform="translate(12.95) scale(.05)" xlink:href="#p"/>
        <use transform="translate(22.95) scale(.05)" xlink:href="#z"/>
        <use transform="translate(31.95) scale(.05)" xlink:href="#y"/>
        <use transform="translate(41.95) scale(.05)" xlink:href="#s"/>
        <use transform="translate(51.95) scale(.05)" xlink:href="#s"/>
        <use transform="translate(61.95) scale(.05)" xlink:href="#o"/>
        <use transform="translate(71.95) scale(.05)" xlink:href="#H"/>
        <use transform="translate(77.9) scale(.05)" xlink:href="#t"/>
        <use transform="translate(82.9) scale(.05)" xlink:href="#r"/>
        <use transform="translate(92.9) scale(.05)" xlink:href="#A"/>
      </g>
      <g id="g">
        <use transform="scale(.05)" xlink:href="#v"/>
        <use transform="translate(5) scale(.05)" xlink:href="#p"/>
        <use transform="translate(15) scale(.05)" xlink:href="#D"/>
        <use transform="translate(25) scale(.05)" xlink:href="#q"/>
        <use transform="translate(34) scale(.05)" xlink:href="#t"/>
        <use transform="translate(39) scale(.05)" xlink:href="#E"/>
        <use transform="translate(42.95) scale(.05)" xlink:href="#F"/>
        <use transform="translate(51.95) scale(.05)" xlink:href="#r"/>
      </g>
      <path fill="#333" d="M233-177c-1 41-23 64-60 70L243 0h-38l-65-103H63V0H30v-248c88 3 205-21 203 71zM63-129c60-2 137 13 137-47 0-61-80-42-137-45v92" id="I"/>
      <g id="h">
        <use transform="scale(.05)" xlink:href="#I"/>
        <use transform="translate(12.95) scale(.05)" xlink:href="#r"/>
        <use transform="translate(22.95) scale(.05)" xlink:href="#s"/>
        <use transform="translate(32.95) scale(.05)" xlink:href="#o"/>
      </g>
      <path fill="#333" d="M106-169C34-169 62-67 57 0H25v-261h32l-1 103c12-21 28-36 61-36 89 0 53 116 60 194h-32v-121c2-32-8-49-39-48" id="J"/>
      <g id="i">
        <use transform="scale(.05)" xlink:href="#J"/>
        <use transform="translate(10) scale(.05)" xlink:href="#D"/>
        <use transform="translate(20) scale(.05)" xlink:href="#z"/>
      </g>
      <g id="j">
        <use transform="scale(.05)" xlink:href="#H"/>
        <use transform="translate(5.95) scale(.05)" xlink:href="#r"/>
        <use transform="translate(15.95) scale(.05)" xlink:href="#D"/>
        <use transform="translate(25.95) scale(.05)" xlink:href="#q"/>
        <use transform="translate(34.95) scale(.05)" xlink:href="#J"/>
        <use transform="translate(44.95) scale(.05)" xlink:href="#r"/>
        <use transform="translate(54.95) scale(.05)" xlink:href="#A"/>
      </g>
      <use transform="scale(.05)" xlink:href="#D" id="k"/>
      <path fill="#333" d="M24 0v-261h32V0H24" id="K"/>
      <g id="l">
        <use transform="scale(.05)" xlink:href="#z"/>
        <use transform="translate(9) scale(.05)" xlink:href="#t"/>
        <use transform="translate(14) scale(.05)" xlink:href="#D"/>
        <use transform="translate(24) scale(.05)" xlink:href="#C"/>
        <use transform="translate(34) scale(.05)" xlink:href="#K"/>
        <use transform="translate(37.95) scale(.05)" xlink:href="#r"/>
      </g>
      <g id="m">
        <use transform="scale(.05)" xlink:href="#z"/>
        <use transform="translate(9) scale(.05)" xlink:href="#t"/>
        <use transform="translate(14) scale(.05)" xlink:href="#D"/>
        <use transform="translate(24) scale(.05)" xlink:href="#t"/>
        <use transform="translate(29) scale(.05)" xlink:href="#r"/>
      </g>
    </defs>
  </g>
</svg>

```

I'm sure some manual cleanup and IDs/comments could make the source even more hackable :)

**Comment by HoverBaum** [comment]
Thanks @waldyrious thats a little improvement I am happy to take 🙂

**Comment by jantman** [comment]
Sorry for the delay, but thanks so much for this! And to @waldyrious for cleaning up the SVG!

**Comment by jantman** [comment]
This is live now, thank you again so much!

**Pull request merged**